packages/ui/components/head-seo/head-seo.test.tsx

```diff
@@ -149,12 +146,4 @@ describe("Tests for HeadSeo component", () => {
       expect(mockedNextSeoEl?.getAttribute("image")).toContain("constructAppImage");
     });
   });
-
-  test("Should render with meeting props", async () => {
```

Comment:
- removing this because meeting prop is removed from `HeadSeo`.
- `HeadSeo` will be completely removed once we completely migrate to App Router